lib/players/human.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../player.rb'
+
+class Human < Player
+
+  def move(token, board)
+    puts "Where would you like to move?"
+    move = gets.chomp
+    move
+  end
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(token, board)
  puts "Where would you like to move?"
  gets.chomp
end
```